nirc_ehr/resources/queries/study/deaths.query.xml

```diff
@@ -18,10 +18,23 @@
                         </fk>
                     </column>
                     <column columnName="remark"/>
-                    <column columnName="performedby"/>
+                    <column columnName="performedby">
+                        <fk>
+                            <fkDbSchema>core</fkDbSchema>
+                            <fkTable>Users</fkTable>
+                            <fkColumnName>UserId</fkColumnName>
+                            <fkDisplayColumnName>DisplayName</fkDisplayColumnName>
+                        </fk>
+                    </column>
                     <column columnName="project">
```

Comment:
We shouldn't be using project in any of these datasets. If they're in the datset we should remove it. The project is what the animal is assigned to.

Reply:
Project removed with required updates